docs/tutorials/intro.md

```diff
@@ -200,19 +204,54 @@ Here's an internationalized date!
 
 # 7. Advanced Data Management
 
-## Using data from the filesystem
+## Generating data
 
-If you have ICU4X data on the file system in a JSON format, it can be loaded via `FsDataProvider`, which needs the `icu_provider_fs` crate.
+Unless you're happy shipping your app with the ~10 locales supported by `icu_testdata`, you will need generate your own data that is filtered to suit your needs.
 
-This provider performs deserialization so is a `BufferProvider`. This means that the feature `"serde"` needs to be enabled on `icu` (or the specific `icu_foo` component crate), and `"deserialize_json"` needs to be enabled on `icu_provider`. There are also `"deserialize_postcard_1"` and `"deserialize_bincode_1"` features available.
+We're going to use the `icu_datagen` crate to do this. It pulls in [CLDR data](http://cldr.unicode.org/index/downloads) and Unicode property data from ICU4C releases to generate the data.
 
-The whole thing can be done by running:
+The `icu_datagen` crate has a command line interface as well as a Rust API, which can be used in `build.rs` files. Here we will use the CLI.
 
+```
+cargo run -p icu_datagen --features bin -- \
+    --cldr-tag latest \
+    --icuexport-tag latest \
+    --out my-data \
+    --format blob \
+    --all-keys --all-locales
+```
+
+The last command is a bit dense, so let's dissect it.
+
+* First, we call `cargo run` which runs a binary
+* We tell it that the crate is named `icu_datagen`
+* We use `--features bin` to be able to build the binary (otherwise the crate will only contain the library)
+* Then we use `--` to separate arguments to `cargo` from arguments to `icu_datagen`
+* `--cldr-tag` informs the program which CLDR version to use
+* `--icuexport-tag` informs the program which ICU-exported data version to use
+* `--out` is the location where we want the generated ICU4X data to be stored
+* `--format blob` to set the format of the output (we'll discuss formats later)
```

Comment:
if we're doing this we should also do `--syntax postcard` imo

Reply:
It's the default and I don't want to overcomplicate it

Reply:
Actually it's not just the default, the `syntax` flag has no effect for blob